app/models/notification.rb

```diff
@@ -8,4 +8,11 @@ class Notification < ApplicationRecord
   belongs_to :room, optional: true
   belongs_to :message, optional: true
 
+  def checked_true_or_false
+    if visiter_id == visited_id
```

Comment:
ここもメソッドに切り出したいですね

```suggestion
    if xxx?
```

```Ruby
def xxx?
  visiter_id == visited_id
end
```

処理の内容を細かく把握していないので、適切なメソッド名を考えるのを諦めてしまいました。すみません。メソッド名は何か良い感じのものを考えてください。

Reply:
a3d9ef6
メソッドの切り出しを行いました。